src/main/java/com/artipie/PermissionsYaml.java

```diff
@@ -52,23 +53,29 @@
      * Ctor.
      * @param conf Config file
      */
-    public RpPermissions(final File conf) {
+    public PermissionsYaml(final File conf) {
         this(readYaml(conf));
     }
 
     /**
      * Ctor.
      * @param yaml Configuration yaml
      */
-    public RpPermissions(final YamlMapping yaml) {
+    public PermissionsYaml(final YamlMapping yaml) {
         this.yaml = yaml;
     }
 
     @Override
     public boolean allowed(final String name, final String action) {
         final YamlMapping all = this.yaml.yamlMapping("permissions");
-        return check(all.yamlSequence(name), action)
-            || check(all.yamlSequence(RpPermissions.WILDCARD), action);
+        final boolean res = check(all.yamlSequence(name), action)
+            || check(all.yamlSequence(PermissionsYaml.WILDCARD), action);
+        if (res) {
+            Logger.info(this, "Operation '%s' allowed for '%s'", action, name);
+        } else {
+            Logger.info(this, "Operation '%s' denied for '%s'", action, name);
```

Comment:
@g4s8 logging level looks to be too high for this kind of messages as they explain they do not log actions, but explain reason why certain actions were made. I'd suggest to decrease logging level to `debug` or make it configurable